This patch fixes quote nesting in format strings in a way that's compatible with
Python 3.11.  A larger cleanup patch [1] applies some of these hunks upstream.

[1] https://codeberg.org/valos/Komikku/commit/e76766c6012d9d5c18c2c3bbaa7dfc11a6ee2d77

diff --git a/komikku/card/tracking.py b/komikku/card/tracking.py
index ce333780..76119e20 100644
--- a/komikku/card/tracking.py
+++ b/komikku/card/tracking.py
@@ -279,7 +279,7 @@ class TrackerRow(Adw.ExpanderRow):
             self.set_expanded(True)
             self.set_arrow_visible(True)
             self.btn.set_visible(False)
-            self.action_row.set_title(f'<a href="{self.tracker.get_manga_url(data['id'])}">{html_escape(data["name"])}</a>')
+            self.action_row.set_title(f'<a href="{self.tracker.get_manga_url(data["id"])}">{html_escape(data["name"])}</a>')
 
             with self.chapters_progress_row.handler_block(self.num_chapter_changed_handler_id):
                 adj = Gtk.Adjustment(
 

